README.md

```diff
@@ -1 +1,112 @@
-# capstone-project
\ No newline at end of file
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+
+# 📖 [Construction Engineering summit website]
+
+**[Construction Engineering summit]** is a website that shows detalais of constructors summit, it has much infomation and it is easy to use it.
+
+## 🛠 Built With
+
+### Tech Stack
+
+Tech Stack
+HTML
+CSS
+JS
+Git/GitHub
+node.js
+Boostrap 5
+
+### Key Features
+
+- **[A webpage that gives you all the information of the owner]**
+
+## 🚀 Live Demo
+
+- [https://ntimanu.github.io/capstone-project/]
+
+## 💻 Getting Started
+
+When you click on the page, the browser shows you the home page of the website.
+
+### Prerequisites
+
+Clone this repository to your desired folder and open html file.
+
+### Setup
+
+Clone this repository to your desired folder:
+Example commands:
+
+```sh
+  cd capstone-project
+  git clone https://github.com/ntimanu/capstone-project
+```
+
+### Install
+
+Install this project with:
+
+<!--
+Example command:
+```sh
+  cd Capstone-1-Project
+  npm install
+```
+--->
+
+### Usage
+
+To run the project, execute the following command:
+
+<!--
+Example command:
+```sh
+  open live server
+```
+--->
+
+## 👥 Authors
+
+👤 **Emmanuel Ntihinyuka**
+
+- GitHub: [@ntimanu](https://github.com/ntimanu)
+- Twitter:[@emmantihinyuka](https://twitter.com/emmantihinyuka)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/ntihinyuka-emmanuel-511890104/)
+
+## 🔭 Future Features
+
+- [ ] **[Add more forms]**
+- [ ] **[Add more pages with information]**
+- [ ] **[Add more Javascript]**
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## ⭐️ Show your support
+
+If you like this project feel free to share with your friends.
+
+## 🙏 Acknowledgments
+
+I would like to thank Microverse for this amazing project.
+and thanks to Caleb.
+
```

Comment:
- [ ] Please input the proper link in the acknowledgement section to shin Cindy to her profile and her design. as seen below.

I want to thank [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), who provided the behance design of the template

Reply:
Done